tools/prerequisites/k8s-setup.sh

```diff
@@ -84,7 +84,7 @@ install_pre_requisites() {
     done
 }
 
-install_container() {
```

Comment:
We may not change the function name
1, It will also install the docker related tools for the whole functionalities
2. It will leave the space for the future updates.

Reply:
It makes sense, but I think the name `install_container` is a little confusing.

Reply:
container means the container runtiome for k8s, which can include docker, containerd, crio